sirepo/job_supervisor.py

```diff
@@ -1078,6 +1078,9 @@ async def _receive_api_statelessCompute(self, req):
         return await self._send_op_analysis(req, "stateless_compute")
 
     def _is_sbatch_login_ok(self, req):
+        r = req.content.get("jobRunMode", self.db.jobRunMode)
+        if r not in sirepo.simulation_db.JOB_RUN_MODE_MAP:
```

Comment:
We have a similar check in [job_api.py](https://github.com/radiasoft/sirepo/blob/c35b955781b998dd8073b33803e1e3da69293269/sirepo/job_api.py#L367). Can the fix be put there?

Reply:
I moved that code to an assertion in job_api. It shouldn't get triggered because the fix on the client makes sure it doesn't call sbatch related methods if it is not enabled.